src/Robo/Common/ComposerMunge.php

```diff
@@ -33,6 +33,12 @@ public static function munge($file1, $file2) {
       $output['require-dev'] = (object) $output['require-dev'];
     }
 
+    // Remove the existence of the new project identifier from invoking the
+    // create-project again in subsequent update calls.
+    if (isset($output['extra']['blt']['new'])) {
+      unset($output['extra']['blt']['new']);
```

Comment:
This doesn't really seem to belong inside this method. Perhaps we should move the file loading and writing functionality out of this method and simply accept and return an array.

We can then set this value and write to file here: https://github.com/acquia/blt/blob/d7c0e4616769c0a3c4a10cf9200491796d4faf30/src/Robo/Commands/Blt/UpdateCommand.php#L338